app/mailers/teacher_mailer.rb

```diff
@@ -56,6 +57,15 @@ def further_information_requested
     )
   end
 
+  def further_information_reminder(due_date:)
```

Comment:
I think I prefer passing this argument in here rather than as an argument to `.with(` but it's inconsistent with the `further_information_request` argument. I'll probably change it to be consistent 🤷‍♂️

Reply:
My understanding is that we have to use `.with` if we want to use `deliver_later` as the parameters get serialised and then the email sending job calls `TeacherMailer.with(deserialised_params).further_information_reminder`.

Reply:
Ah right. That makes sense. I did not know that. I'll change it.